CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# How to contribute?
+## Pre-requisites
+Open a free account on Naas Cloud so we can test the templates in a similar environment.
```

Comment:
Link to the respective URLs:
- Naas cloud
- Contributor program registration
- GitHub token creation

This will make easier to redirect.